src/dss/create_notebook.py

```diff
@@ -79,19 +83,16 @@ def create_notebook(name: str, image: str, lightkube_client: Client) -> None:
     try:
         k8s_resource_handler.apply()
 
-        wait_for_deployment_ready(lightkube_client, namespace=DSS_NAMESPACE, deployment_name=name)
+        wait_for_deployment_ready(
+            lightkube_client, namespace=DSS_NAMESPACE, deployment_name=name, timeout_seconds=None
+        )
 
         logger.info(f"Success: Notebook {name} created successfully.")
     except ApiError as err:
         logger.debug(f"Failed to create Notebook {name}: {err}.", exc_info=True)
         logger.error(f"Failed to create Notebook with error code {err.status.code}.")
         logger.info(" Check the debug logs for more details.")
-        raise RuntimeError()
-    except TimeoutError as err:
-        logger.debug(f"Failed to create Notebook {name}: {err}", exc_info=True)
-        logger.error(f"Timed out while trying to create Notebook {name}.")
-        logger.warn(" Some resources might be left in the cluster.")
-        logger.info(" Check the status with `dss list`.")
+        k8s_resource_handler.delete()
```

Comment:
Can we use `remove_notebook` here instead of the `krh.delete()`?  `remove_notebook` "owns" how notebooks should be removed, and it has logic to log errors and such.  It also was implemented without [adding labels](https://github.com/canonical/data-science-stack/pull/124/files#diff-45664d89958707fc62dad665229f032fb4350c6f707bb2af885309d7d692b807R76) like done above - nothing wrong with the labels, but a different method was used so we should be consistent here.